app/src/main/java/org/kiwix/kiwixmobile/webserver/ZimHostActivity.java

```diff
@@ -205,8 +205,7 @@ void toggleHotspot() {
     if (isServerStarted) {
       ip = getAddress();
       ip = ip.replaceAll("\n", "");
-      serverTextView.setText(
-          getString(R.string.server_started_message) + " " + ip);
+      serverTextView.setText(getString(R.string.server_started_message) + " " + ip);
```

Comment:
You can put format args in string resources
```
<string name="server_started_message">Enter this ip address into your browser to access the server %s</string>
```
and then use `getString(R.string.server_started_message, ip)`

Reply:
Fixed. 50e072bf18263f511b00025af6a44e71a17f536b